src/build.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
 
     zfs_pipeline:function() {
         var zfs = {
-            url: "https://gitlab.openebs.ci/api/v4/projects/17/pipelines",
+            url: "https://gitlab.openebs.ci/api/v4/projects/17/pipelines?ref=zfs-0.7-release",
```

Comment:
little nitpicking here, why are we hardcoding branch name here. Isn't there a way to pass during runtime, because in this case, every time a new release will come code has to be modified, which is not a good idea IMO!

Reply:
Right @harshvkarn I also thinking about this earlier but not yet implemented! @bhaskarhc Can you look how to pass the branch name using environment variable during build time